.gitignore

```diff
@@ -2,3 +2,4 @@
 *.gem
 
 /debug.log
+/gemfiles/*.lock
```

Comment:
Why do you want to ignore that? 
You had gemfile.lock generated when runnning the tests?